### DISCUSSION TITLE
Only listening to localhost and docker

### DISCUSSION THREAD

**Discussion by funk01** [AdguardTeam/AdGuardHome#4582] [Q&A]
Hi there,

I can´t figure out why, but AdGuard only listens to:

127.0.0.1
172.17.0.4

WebGui works fine, but i cant manage to see him the IP-range of my net. It´s installed on a Synology NAS and inside a Docker.
The protocol looks good, but it doesnt show IP of my NAS eg. 192.168.178.xx - only 127.0.0.1 and the 172.17.0.4.
How can it listen to 192.168.178.xx....? 

2022/05/22 11:09:24.570388 [info] AdGuard Home, version v0.107.6
2022/05/22 11:09:24.570512 [info] This is the first time AdGuard Home is launched
2022/05/22 11:09:24.570556 [info] Checking if AdGuard Home has necessary permissions
2022/05/22 11:09:24.575101 [info] AdGuard Home can bind to port 53
2022/05/22 11:09:24.577212 [info] Initializing auth module: /opt/adguardhome/work/data/sessions.db
2022/05/22 11:09:24.752939 [info] auth: initialized.  users:0  sessions:0
2022/05/22 11:09:24.753030 [info] Initialize web module
2022/05/22 11:09:24.753236 [info] This is the first launch of AdGuard Home, redirecting everything to /install.html 
2022/05/22 11:09:24.753352 [info] AdGuard Home is available at the following addresses:
2022/05/22 11:09:24.753746 [info] Go to http://127.0.0.1:3000
2022/05/22 11:09:24.753787 [info] Go to http://172.17.0.4:3000
2022/05/22 11:22:01.812845 [info] Starting the DNS proxy server
2022/05/22 11:22:01.812924 [info] Ratelimit is enabled and set to 20 rps
2022/05/22 11:22:01.812945 [info] The server is configured to refuse ANY requests
2022/05/22 11:22:01.812967 [info] DNS cache is enabled

THANKS!
AF

**Comment by ainar-g** [comment]
Those are the addresses of the virtual interfaces that a Docker container can access by default.  See <https://docs.docker.com/network/network-tutorial-standalone/> and generally the Docker documentation on networking.
